TypeCobol/Compiler/Nodes/Statement.cs

```diff
@@ -172,6 +172,15 @@ public override bool VisitNode(IASTVisitor astVisitor)
         }
     }
 
+    public class ExitPerform : GenericNode<ExitPerformStatement>, Statement
```

Comment:
Not required but could be useful: add a computed property for `Cycle`.

```csharp
public bool Cycle => CodeElement.Cycle?.Value ?? false;
```

or something similar...

Reply:
Done